R/build-home.R

```diff
@@ -127,6 +127,7 @@
 #' The example below creates a sidebar whose only components will be the
 #' authors section, a custom section, and a Dev Status section if there are
 #' badges.
+#' The `text` can be Markdown or HTML.
```

Comment:
I think you can just the text is treated as markdown (and hence HTML will be preserved)